src/controllers/RaceController.js

```diff
@@ -0,0 +1,51 @@
+import { Random } from "@woowacourse/mission-utils";
+
+class RaceController {
+
+  createPlayersPositions(playerNumber) {
+    const positions = Array(playerNumber).fill(0);
+    return positions;
```

Comment:
```suggestion
    return Array(playerNumber).fill(0);
```
positions가 메서드 내부에서 다른 곳에 쓰이지 않고 바로 return 되므로, 상수 선언 대신 바로 return 하는 방식은 어떠신가요? 🤔